docs/changelog.rst

```diff
@@ -5,6 +5,10 @@ CHANGELOG
 2.87.2+dev (XXXX-XX-XX)
 -----------------------
 
+
+**Minor improvements**
+- Set begin_date not null for touristic events
+
```

Comment:
Add 

Bug fix : 
TouristicEvents with no end dates are not returned in APIv2

Reply:
With a reference to the dedicated issue (#3127)